src/shared/proto/messages_robocup_ssl_detection_tracked.proto

```diff
@@ -0,0 +1,122 @@
+syntax = "proto2";
+
+message Vector2 {
+    required float x = 1;
+    required float y = 2;
+}
+
+message Vector3 {
+    required float x = 1;
+    required float y = 2;
+    required float z = 3;
+}
+
+// The team color of the robot
+enum TeamColor {
+    UNKNOWN = 1;
+    YELLOW = 2;
+    BLUE = 3;
+}
+
+message RobotId {
+    // The robot number
+    required uint32 id = 1;
+    // The team color
+    required TeamColor team_color = 2;
+}
+
+enum ChipKickDetectionSupport {
+    UNSUPPORTED = 0;
+    FIRST_TOUCHDOWN_ONLY = 1;
+    ALL_TOUCHDOWNS = 2;
+}
+
+// A single tracked ball
+message TrackedBall {
+    // The quality of the detection
+    // A value between 0 (not visible) and 1 (visible)
+    // The exact implementation depends on the source software
+    optional float quality = 1;
+
+    // The position (x, y, height) [m] in the ssl-vision coordinate system
+    required Vector3 pos = 2;
+
+    // The velocity [m/s] in the ssl-vision coordinate system
+    optional Vector3 vel = 3;
+
+    // The level of chip kick detection support by the source implementation
+    required ChipKickDetectionSupport chip_kick_detection_support = 4;
```

Comment:
Would recommend to move this to a general "capabilities" field in the TrackerWrapperPacket to indicate what the sending software supports. No need to have it in every TrackedBall.

Reply:
without the touch down locations, this is not needed anymore. If we need this later, I like the idea of the capability list.

Reply:
Actually, there are some interesting capabilities. I've added them like you suggested.